orthanc/orthanc-anon/config/orthanc.json

```diff
@@ -1,4 +1,22 @@
 {
+  "ExtraMainDicomTags" : {
+    "Instance": [
+      "Rows",
+      "Columns",
+      "ImageType",
+      "SOPClassUID",
+      "ContentDate",
+      "ContentTime",
+      "FrameOfReferenceUID",
+      "PatientID",
+      "PatientName",
+      "AccessionNumber",
+      "PixelSpacing",
+      "SpecificCharacterSet",
+      "BitsAllocated",
+      "BitsStored"
+    ]
+  },
```

Comment:
Oh what's the reasoning for having these here?

Reply:
Well, it passes the system test without it. I can't remember if I was trying to fix something which I fixed another way, or thought it was needed for some other reason. This list was suggested somewhere in the orthanc docs ISTR. Will look into it.